src/DataMiner/SLNetTypes/Automation/CustomEntryPoint/AutomationEntryPoint.cs

```diff
@@ -90,6 +90,12 @@ public enum Types
 			/// </summary>
 			OnDomAction = 10,
 
+			/// <summary>
+            /// An OnApiTrigger is defined with:
```

Comment:
Can maybe be made: 'An OnApiTrigger "entry point" is defined' to be similar to the some other ones.